addons/web/static/src/scss/list_view.scss

```diff
@@ -101,6 +101,9 @@
         .o_list_record_selector {
             width: 40px;
             padding-left: $o-horizontal-padding;
+            .custom-control-label {
```

Comment:
this element has a width of 0px, so I don't get why you added this rule. Can we remove it?

Reply:
ok nevermind :D